runtime/request.go

```diff
@@ -17,10 +32,14 @@ type Request struct {
 
 	// APICall specific parameters.
 	// These will be empty for operations with a type not APICall
+	API        *APIDesc   // Metadata about the API endpoint being called
 	Service    string     // Which service is processing this request
-	Endpoint   string     // Which API endpoint was called.
-	Path       string     // What was the path made to the API server.
+	Endpoint   string     // Which API endpoint is being called
+	Path       string     // What was the path made to the API server
 	PathParams PathParams // If there are path parameters, what are they?
+	// Payload is the decoded request payload, or nil if the endpoint
+	// has no request payload or the endpoint is raw.
+	Payload any
```

Comment:
We should set this for Pubsub requests too.

Reply:
Aren't we already? I thought I added that